cache/cloudfront_prod.tf

```diff
@@ -74,16 +74,15 @@ resource "aws_cloudfront_distribution" "wellcomecollection_org" {
         "cachebust",
         "current",
         "page",
-        "result",
-        "toggle",
-        "uri",
-
         # This is used to fetch articles client-side, e.g. related stories.
         # When it's missing, we may show the wrong stories as "read this next"
         # on articles.
         #
         # See https://github.com/wellcomecollection/wellcomecollection.org/issues
         "params",
+        "result",
```

Comment:
Minor nit: are these tied to the comment above? If not, can we get a newline to make it clear which parameters are part of the comment?

Also, that comment is hilari-bad, I meant to link to https://github.com/wellcomecollection/wellcomecollection.org/issues/7461

Reply:
Yeah, will sort these out